Libraries/LLM/Llama.swift

```diff
@@ -253,3 +253,20 @@ public struct LlamaConfiguration: Codable {
 
     }
 }
+
+// MARK: - LoRA
```

Comment:
This is how we add the introspection required to apply the LoRA adapter.

- [ ] document this